kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/HowAreYouUsingKolibri.vue

```diff
@@ -36,18 +36,21 @@
     mixins: [commonSyncElements],
     inject: ['wizardService'],
     data() {
-      const selected = this.wizardService.state.context['onMyOwnOrGroup'] || UsePresets.ON_MY_OWN;
+      const selected = this.wizardService.state.context['onMyOwnOrGroup'] || Presets.PERSONAL;
       return {
         selected,
       };
     },
     computed: {
       isOnMyOwnSetup() {
-        return this.selected === UsePresets.ON_MY_OWN;
+        return this.selected === Presets.PERSONAL;
       },
       UsePresets() {
         return UsePresets;
       },
+      Presets() {
+        return Presets;
+      },
```

Comment:
I noticed an error in the console while going through the on my own flow:

```
vuex.esm.js:460 [vuex] unknown mutation type: SET_FACILITY_PRESET
commit @ vuex.esm.js:460
boundCommit @ vuex.esm.js:409
handleContinue @ HowAreYouUsingKolibri.vue:60
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
click @ OnboardingStepBase.vue:200
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
invokeWithErrorHandling @ vue.runtime.esm.js:1863
Vue.$emit @ vue.runtime.esm.js:3903
handleClick @ KButton.vue:181
invokeWithErrorHandling @ vue.runtime.esm.js:1863
invoker @ vue.runtime.esm.js:2188
original._wrapper @ vue.runtime.esm.js:6961
```

Looks like the call to the `SET_FACILITY_PRESET` mutation below is calling a non-existent mutation?